### DISCUSSION TITLE
Support for Mastodon embeds via Liquid

### DISCUSSION THREAD

**Discussion by andypiper** [forem/forem#18709] [Feature Requests]
With much of the Twittersphere moving to the Fediverse, and posts about this on DEV, it would be fantastic if we could directly embed Toots into our posts.

What could we do to help move this along? Mastodon supports oEmbed already, so that's a useful building block.

**Comment by thomasbnt** [comment]
I still remember when we had the possibility to add a link from our Mastodon on our DEV profile. Could be great with embed! 🙏🏼

**Comment by J3ffJessie** [comment]
100% support this and would love to see how this can be integrated.

**Comment by michael-tharrington** [comment]
OOo right on! I think this is a great idea and am all for it. I know Mastodon has been on the mind... I'll raise this one to the team.

**Comment by haubles** [comment]
Hey! I am working on a post that includes a handful of Mastodon toots and was wondering if this might be coming soon? Very excited this may be in the works already!
